test/integration/virtual-rules/aria-braille-equivalent.js

```diff
@@ -0,0 +1,61 @@
+describe('aria-braille-equivalent virtual-rule', () => {
+  afterEach(() => {
+    axe.reset();
+  });
```

Comment:
I believe this isn't necessary 

```suggestion
```

Reply:
Configuration isn't reverted until axe.reset() is called. It's not in testutils. Where would you think this might happen?